rocketpy/control/controller.py

```diff
@@ -0,0 +1,132 @@
+from ..prints.controller_prints import _ControllerPrints
+
+
+class _Controller:
+    """A class for storing and running controllers on a rocket. Controllers
+    have a controller function that is called at a specified sampling rate
+    during the simulation. The controller function can access and modify
+    the objects that are passed to it. The controller function also stores the
+    variables of interest in the objects that are passed to it."""
+
+    def __init__(
+        self,
+        interactable_objects,
+        controller_function,
```

Comment:
This is already the `Controller` class, do we really need to point that this is the `controler` function?

Of course, letting an argument to be named as `function` is also not a good idea since this is one special variable name in python.

My suggestion is to change this argument to `func`, which is gives the same understandment to the user while being shorter.

Reply:
Personally I believe `func` is a bit of a too vague name, of course this is just a nomenclature debate (things are clarified in the description), but my suggestion would be `control_function`.

Reply:
I get that repeating the name controller can be a little messy, but I think the clarity we gain from it is pretty useful.

I could see this being named  `callback_function`, but honestly I just prefer `controller_function`

Reply:
Ok, guess I've lost this one kids,
but no problem